helm/assets/dashboards/overview.json

```diff
@@ -0,0 +1,176 @@
+        "query": "source:helm datacenter:$datacenter.value kube_cluster_name:$kube_cluster_name.value helm_release:$helm_release.value helm_status:$helm_status.value helm_namespace:$helm_namespace.value",
+        "tags_execution": "and",
+        "event_size": "l"
+      },
+      "layout": {
+        "x": 5,
+        "y": 0,
+        "width": 7,
+        "height": 3
+      }
+    },
+    {
+      "id": 5229210289262404,
+      "definition": {
+        "title": "Helm releases",
+        "title_size": "16",
+        "title_align": "left",
+        "show_legend": true,
+        "legend_layout": "auto",
+        "legend_columns": [
+          "avg",
+          "min",
+          "max",
+          "value",
+          "sum"
+        ],
+        "type": "timeseries",
+        "requests": [
+          {
+            "formulas": [
+              {
+                "formula": "query1"
+              }
+            ],
+            "response_format": "timeseries",
+            "queries": [
+              {
+                "query": "sum:helm.release{$datacenter,$kube_cluster_name,$helm_release,$helm_status,$helm_namespace} by {helm_status}",
+                "data_source": "metrics",
+                "name": "query1"
+              }
+            ],
+            "style": {
+              "palette": "dog_classic",
+              "line_type": "solid",
+              "line_width": "normal"
+            },
+            "display_type": "bars"
+          }
+        ]
+      },
+      "layout": {
+        "x": 0,
+        "y": 3,
+        "width": 12,
+        "height": 3
+      }
+    },
+    {
+      "id": 8212990758383134,
+      "definition": {
+        "title": "Helm releases (only shows 500)",
+        "title_size": "16",
+        "title_align": "left",
+        "type": "query_table",
+        "requests": [
+          {
+            "formulas": [
+              {
+                "formula": "query1",
+                "conditional_formats": [],
+                "limit": {
+                  "count": 500,
+                  "order": "desc"
+                },
+                "cell_display_mode": "bar"
+              }
+            ],
+            "response_format": "scalar",
+            "queries": [
+              {
+                "query": "sum:helm.release{$datacenter,$kube_cluster_name,$helm_release,$helm_status,$helm_namespace,$helm_namespace} by {helm_chart_name,helm_chart_version,cluster_name,datacenter,helm_status,helm_revision,helm_release,helm_namespace}",
+                "data_source": "metrics",
+                "name": "query1",
+                "aggregator": "avg"
+              }
+            ]
+          }
+        ],
+        "has_search_bar": "auto"
+      },
+      "layout": {
+        "x": 0,
+        "y": 6,
+        "width": 12,
+        "height": 7
+      }
+    }
+  ],
+  "template_variables": [
+    {
+      "name": "datacenter",
+      "default": "*",
+      "prefix": "datacenter",
+      "available_values": []
+    },
```

Comment:
`datacenter` is a custom tag - we cannot assume that our customers have it

Reply:
Thanks. I added a new commit that removes it :+1: